### PR TITLE
[github](action) optimize trigger teamcity pipeline

### PR DESCRIPTION
**Pull request by hello-stephen**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hello-stephen** [comment]
run buildall

**Comment by hello-stephen** [comment]
run buildall

**Review by englefly — DISMISSED** [review]

**Review by shuke987 — APPROVED** [review]

File: .github/workflows/comment-to-trigger-teamcity.yml [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+            "${COMMENT_BODY}" == *'run compile'* ||
+            "${COMMENT_BODY}" == *'run beut'* ||
+            "${COMMENT_BODY}" == *'run feut'* ||
+            "${COMMENT_BODY}" == *'run p0'* ||
+            "${COMMENT_BODY}" == *'run p1'* ||
+            "${COMMENT_BODY}" == *'run external'* ||
+            "${COMMENT_BODY}" == *'run pipelinex_p0'* ||
+            "${COMMENT_BODY}" == *'run clickbench'* ||
+            "${COMMENT_BODY}" == *'run arm'* ||
+            "${COMMENT_BODY}" == *'run tpch'* ]]; then
```
[the remark this answers is not in the document]
Comment (hello-stephen) [reply .github/workflows/comment-to-trigger-teamcity.yml:50]: keep

**Comment by hello-stephen** [comment]
run buildall

**Comment by hello-stephen** [comment]
run buildall

**Review by mrhhsg — APPROVED** [review]
LGTM

**Review by englefly — APPROVED** [review]

**Pull request merged**